Gems/Multiplayer/Code/Source/Editor/MultiplayerEditorSystemComponent.cpp

```diff
@@ -188,6 +189,11 @@ namespace Multiplayer
                 console->PerformCommand("disconnect");
             }
 
+            if (m_connectionEvent.IsScheduled())
+            {
+                m_connectionEvent.RemoveFromQueue();
```

Comment:
It is safe to remove from queue without checking.

Reply:
Good question! IsScheduled is checking if it’s queued to schedule, although technically the check isn’t needed.